File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    a = s.split()
    b = [int(x) for x in a]
    
    max = b[0]
    min = b[0]
    
    for i in range(len(a)):
        if b[i] > max:
            max = b[i]
        if b[i] < min:
            min = b[i]

    answer = '{} {}'.format(min, max)
    return answer

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String[] str = s.split(" ");    
        
        int[] nums = new int[str.length];
        
        for(int i=0; i<str.length; i++){
            nums[i] = Integer.parseInt(str[i]);
        }
        
        
        int min = nums[0];
        int max = nums[0];
        
        for(int i=1; i<str.length; i++){
            if(nums[i] < min){
                min = nums[i];
            }
            
            if(nums[i] > max){
                max = nums[i];
            }
        }
        
        return min + " " + max;
        
        // Arrays.sort(nums);
        
        // String answer = "";
        // return answer;
    }
}